esti/multipart_test.go

```diff
@@ -54,12 +54,53 @@ func TestMultipartUpload(t *testing.T) {
 	require.Equal(t, partsConcat, getResp.Body, "uploaded object did not match")
 }
 
-func uploadMultipartParts(t *testing.T, logger logging.Logger, resp *s3.CreateMultipartUploadOutput, parts [][]byte, firstIndex int) []*s3.CompletedPart {
+func TestMultipartUploadHostStyleClient(t *testing.T) {
```

Comment:
Why is this test unique to host-style? Shouldn't we test it for path-style too?

Reply:
we do